README.md

```diff
@@ -53,14 +53,13 @@ instance-1  us-central1-a  n1-standard-1               10.240.0.2   8.34.219.29
 
 ### Installing additional components
 
-By default, all gcloud components are installed on the default image: 
+By default, all gcloud components are installed on the default image:  [https://cloud.google.com/sdk/downloads#apt-get](https://cloud.google.com/sdk/downloads#apt-get)
```

Comment:
FYI no need to use markdown syntax if you're just pasting the link. :) (no need to fix at this time)